splash/lua_modules/wraputils.lua

```diff
@@ -74,6 +74,40 @@ local function raises_async(func)
   end
 end
 
+function print_r ( t )
```

Comment:
there is inspect.lua bundled (see https://github.com/scrapinghub/splash/blob/master/splash/lua_modules/vendor/inspect.lua), and a wrapper for it: https://github.com/scrapinghub/splash/blob/master/splash/lua_modules/repr.lua, maybe they can help